src/Slides.js

```diff
@@ -0,0 +1,17 @@
+import { URL } from 'url';
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  if (!(host === 'slides.com' || host === 'www.slides.com')) return false;
+  if (pathname.includes('/embed')) return false;
+
+  const pathTrimmed = pathname.split('/').filter(Boolean);
+  if (pathTrimmed.length !== 2) return false;
+  return true;
+};
+
+export const getHTML = url => {
+  const _url = new URL(url);
```

Comment:
I think we should extract this into a separate method (`getSlidesIFrameSrc`), like we do in the YouTube transformer

https://github.com/MichaelDeBoey/gatsby-remark-embedder/blob/da007b20ef4ea9c5efd4abd3d3d3e73556f15283/src/YouTube.js#L50-L54